css/style.css

```diff
@@ -0,0 +1,293 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  min-width: 100%;
+  margin-top: 70px;
+}
+
+header > div {
+  background-color: #f72210;
+}
+
+.header-btn div {
+  float: right;
+  margin-bottom: 20px;
+}
+
+.main-content div {
+  height: 300px;
+}
+
+header div:first-child {
+  position: relative;
+}
+
+.row.header-top {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- [ ] You should use bootstrap classes `p-*` and `m-*` to set the margins and padding.